dex/msgjson/types.go

```diff
@@ -1041,18 +1041,16 @@ type PreValidateBondResult struct {
 	AssetID   uint32 `json:"assetID"`
 	Amount    uint64 `json:"amount"`
 	Expiry    uint64 `json:"expiry"` // not locktime, but time when bond expires for dex
-	BondID    Bytes  `json:"bondID"`
```

Comment:
Please doc on the `Signature` field that the preimage is the result serialization concatenated with the `RawTx` in the request.